### ISSUE TITLE
Some Suggestions

### ISSUE THREAD

**Issue by linustouchtips** [HausemasterIssue/TickShift#16]
Your explanation is wrong. The client still sends a CPacketPlayer packet every 20 ticks, regardless of whether or not you are moving/rotating. The client doesn't send any CPacketPlayer.Position, CPacketPlayer.Rotation, CPacketPlayer.PositionRotation when you're not moving/rotating, but it still sends the standard CPacketPlayer packet. The idea of having to stand still for this to work is pretty stupid when you could just cancel these packets and prevent them from being sent.

**Comment by HausemasterIssue** [comment]
@linustouchtips if you want to improve it please make a pull request, i'd love you too!

so your saying i just cancel the CPacketPlayer.Position, CPacketPlayer.Rotation etc. packets when i move?

**Comment by linustouchtips** [comment]
Yes, instead of having a movement check, just cancel the CPacketPlayer packet if it it's a rotating or moving packet.

**Comment by arjunnain17** [comment]
wouldn't that be sort of like blink

**Comment by HausemasterIssue** [comment]
yeah, dosent blink just cancel all cpacketplayer packets for a bit lol?

**Comment by Skitttyy** [comment]
actual fucking retard does not even know what blink does LOL

**Comment by HausemasterIssue** [comment]
sorry

**Comment by HausemasterIssue** [comment]
guess im just a fucking retard
